lib/ansible/modules/system/seport.py

```diff
@@ -15,12 +15,12 @@
 module: seport
 short_description: Manages SELinux network port type definitions
 description:
-     - Manages SELinux network port type definitions.
+    - Manages SELinux network port type definitions.
 version_added: "2.0"
 options:
   ports:
     description:
-      - Ports or port ranges, separated by a comma.
+      - Ports or port ranges. Can be a list or comma separated string.
```

Comment:
you should note that 'list' is only accepted since version 2.6

Reply:
Done.